docs/UsersGuide.asciidoc

```diff
@@ -393,12 +393,12 @@ releases, jobs for which long-lasting bug reports exist, etc.
 === Filtering test results and builds ===
 
 At the top of the test results overview page is a form which allows filtering tests by result,
-architecture and TODO-status.
+architecture and TODO-status. TODO-status means that tests still require review.
 
 image::images/filter_form.png[Filter form]
 
 There is also a similar form at the bottom of the index page which allows filtering builds by
-group and customizing the limits.
+group and customizing the limits. This also allows filtering by TODO-status
```

Comment:
This is not correct. The form relates to the filter form at the bottom of the *index* page, e.g. openqa.opensuse.org/ on the bottom. This is not the same as the *tests* page. Better write

```suggestion
group and customizing the limits.
Also the 'All tests' table allows filtering by the TODO-status.
```

Reply:
Oh yeah I should have caught that, seeing as there is neither a similar form and much less one at the bottom of the All Tests page